homeassistant/components/camera/abode.py

```diff
@@ -0,0 +1,103 @@
+"""
+This component provides HA camera support for Abode Security System.
```

Comment:
Next time please keep the PR small. For example, this platform could have been done in a separate PR.

Reply:
I promise to keep them smaller from now on... I've learned my lesson!